api/chaos/experimentation/v1/form.proto

```diff
@@ -0,0 +1,14 @@
+syntax = "proto3";
+
+package clutch.chaos.experimentation.v1;
+
+option go_package = "experimentationv1";
+
+message Form {
```

Comment:
I think it would make more sense to name this 'Properties'/'Property'. Form/TextField seem like UI elements. I don't think that's what these represent?

Reply:
`Properties` sounds good 👍